conftest.py

```diff
@@ -10,3 +10,23 @@
 collect_ignore = ['setup.py']
 
 pytest.register_assert_rewrite("skfda")
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--runslow", action="store_true", default=False, help="run slow tests"
+    )
+
+
+def pytest_configure(config):
+    config.addinivalue_line("markers", "slow: mark test as slow to run")
+
+
+def pytest_collection_modifyitems(config, items):
```

Comment:
Slow tests should be run in Travis, so modify the Travis configuration to take that into account.